crypto/include/internal/ec_int.h

```diff
@@ -9,8 +9,8 @@
 
 /* Internal EC functions for other submodules: not for application use */
 
-#ifndef HEADER_OSSL_EC_INTERNAL_H
-# define HEADER_OSSL_EC_INTERNAL_H
+#ifndef OPENSSL_EC_INTERNAL_H
+# define OPENSSL_EC_INTERNAL_H
```

Comment:
this is also different than the file name, and should probably be OPENSSL_INTERNAL_EC_INT_H